admin-app/image.go

```diff
@@ -10,6 +13,7 @@ import (
 	"github.com/fossoreslp/go-uuid-v4"
 	"github.com/gin-gonic/gin"
 	"github.com/matheusgomes28/urchin/common"
+	"github.com/nfnt/resize"
```

Comment:
Not sure I'd like to use this dependency here for a few reasons:

+ It's not well maintained (the git repo says last changes was nearly a decade ago!).
+ Deviates from the Golang std lib.

Apparently, the `golang.org/x/image` module provides some functionality that can help (`draw.CatmullRom.Scale`). Can you change it to that if possible? This library is maintained by Go developers

Reply:
Actually, `github.com/nfnt/resize` is archived so it's no longer maintained!